BackendApartmentReservation/BackendApartmentReservation/BackendApartmentReservation.csproj

```diff
@@ -55,8 +55,4 @@
     </Content>
   </ItemGroup>
 
-  <ItemGroup>

```

Comment:
Why is this removed from csproj?

Reply:
This must be removed automatically for some reason, added back in